InnoViber.DAL/Repositories/MessageRepository.cs

```diff
@@ -14,4 +14,12 @@ public MessageRepository(ViberContext context) : base(context)
             .Include(x => x.User)
             .Include(x => x.Chat)
             .FirstOrDefaultAsync(ct);
+
+    public new async Task<MessageEntity> Update(MessageEntity entity, CancellationToken ct)
+    {
+        _dbSet.Update(entity);
+        _dbSet.Entry(entity).Property(x => x.Date).IsModified = false;
```

Comment:
why this line's needed?

Reply:
Becaause i don't want allow modifying message date

Reply:
Because i don't want to allow message date change